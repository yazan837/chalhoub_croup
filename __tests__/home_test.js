import React from 'react';
import {create} from 'react-test-renderer';
import Home from '../src/home';

const tree = create(<Home />);
test('test home page ui', () => {
  jest.mock('react-redux');
  expect(tree).toMatchSnapshot();
});
