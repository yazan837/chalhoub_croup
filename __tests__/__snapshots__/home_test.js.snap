// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test home page ui 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flex": 1,
    }
  }
>
  <View>
    <RCTScrollView
      data={Array []}
      getItem={[Function]}
      getItemCount={[Function]}
      horizontal={false}
      keyExtractor={[Function]}
      onContentSizeChange={[Function]}
      onLayout={[Function]}
      onMomentumScrollBegin={[Function]}
      onMomentumScrollEnd={[Function]}
      onScroll={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      removeClippedSubviews={false}
      renderItem={[Function]}
      scrollEventThrottle={50}
      showsHorizontalScrollIndicator={false}
      stickyHeaderIndices={Array []}
      testID="productsList"
      viewabilityConfigCallbackPairs={Array []}
    >
      <View />
    </RCTScrollView>
  </View>
</View>
`;
